crates/integration-tests/tests/gateway/entity_caching/directive_scopes.rs

```diff
@@ -9,6 +9,7 @@ use integration_tests::{
 };
 
 #[test]
+#[ignore] // FIXME: fix cache scopes
```

Comment:
Just commenting to make sure these are intend,d and the deleted hook tests below too